docs/scripts.rst

```diff
@@ -130,6 +130,10 @@ Explanation for the keys:
 - ``urls``: dictionary of table names and the respective urls
 - ``version``: "1.0.0"
 
+``fields``: Although retriever automatically detects the schema of the table,
```

Comment:
Pk-auto should only be used to create a new non existing field. For example, my data is A, B,C
and I want my data to be Z, A,B,C, pk-auto will give Z integers starting from 1 . 
If my data is 
```
A,B,C
1,a,4
2,b,n
```
And A uniquely identifies each column. We can use pk-int since A is all integers. the same applies to pk-char